README.rst

```diff
@@ -37,6 +37,32 @@ Create an Enum-class and pass it as first argument to the Django model EnumField
     class Beer(models.Model):
         style = enum.EnumField(BeerStyle, default=BeerStyle.LAGER)
 
+You can also set default value on your enum class using ``__default__``
+attribute
```

Comment:
I'm not sure I agree on the ambiguity that this introduces. To quote the zen of Python "There should be one-- and preferably only one --obvious way to do it."

What bother me most is that having two ways to declare the default means I'll have to check the `Enum` declaration every time.

Assume the following scenario: I've inherited a code-base, and I'm going through its models and I see:

```python
from .enums import BeerStyle


class Beer(models.Model):
  style = enum.EnumField(BeerStyle)
```

Now I have to check 2 places to find out if there is a default: the field declaration above, and the enum declaration. This every single time the `EnumField` is used.